src/js/claims-status/components/AddFilesForm.jsx

```diff
@@ -109,10 +109,10 @@ class AddFilesForm extends React.Component {
             <div className="document-title-size">
               <div className="document-title-row">
                 <div className="document-title-text-container">
-                  <div className="document-title-header">
-                    <h4 className="title">{file.name}</h4>
+                  <div>
+                    <h4 className="document-title">{file.name}</h4>
```

Comment:
I don't think this needs to be an `h4`.

Reply:
Good point @jbalboni, additional style rules have been provided to `document-title` to mimic the `<h4>` appearance.